_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_data.h

```diff
@@ -783,6 +783,9 @@ namespace Base
         mfxI32              PrevRAP             = -1;
         mfxU16              NumRecode           = 0;
         mfxI8               QpY                 = 0;
+        mfxI8               m_minQP             = 0;
```

Comment:
Hi Hongbo,

May I know if any check for minQP/maxQP in valid range?

Thanks,
Thomas

Reply:
The range is (1, 51).

Reply:
It should be higher than 51 for 10bit.

Reply:
The mediasdk library will add the offset so that it is higher than 51.

Reply:
I mean if there is any implementation to check minQP/maxQP is within this range?

Reply:
Updated with range check in the VA layer.

Reply:
Thanks Hongbo! The check looks good to me. But any chance to put to a common place for Windows and Linux? Any requirement on Windows?

Reply:
Thanks, Thomas! This is from Linux requirement. I have no idea about the Windows requirement.

Reply:
that's OK. let me figure out Windows changes later.

Reply:
@thomasli21801 , please approve if it looks good to you.